code/modules/integrated_electronics/subtypes/medical.dm

```diff
@@ -0,0 +1,668 @@
+
+#define SURGERY_DURATION_DELTA rand(9,11) / 10 // delta multiplier for all surgeries, from 0.9 to 1.1
+#define SURGERY_FAILURE -1
+#define SURGERY_BLOCKED -2
+#define SURGERY_ORGAN_REMOVE 0
+#define SURGERY_ORGAN_INSERT 1
+#define SURGERY_ORGAN_HEAL 2
+#define SURGERY_ORGAN_CONNECT 3
+#define SURGERY_ORGAN_DISCONNECT 4
+#define SURGERY_BONEGEL 1
+#define SURGERY_BONESET 2
+#define SURGERY_BONESET_ULTRA 3
+#define SURGERY_SAW 4
+#define SURGERY_RETRACTOR 5
+#define SURGERY_CAUTERY 6
+#define SURGERY_DRILL 7
+#define SURGERY_FAILED_STATE -1337
+
+/obj/item/integrated_circuit/medical
+	category_text = "Medical"
+
+/obj/item/integrated_circuit/medical/surgery_device
+	name = "basic surgery device" // help, I don't know, how to name this circuit :(
+	desc = "This circuit contains instructions to use medical instruments for body manipulation. Perhaps it does operation like a surgery instrument inserted in it."
```

Comment:
```suggestion
	desc = "This device is a manipulator compatible with most surgical instruments. While holding such an instrument, it can perform various surgical operations."
```